platform/mellanox/mlnx-sai.mk

```diff
@@ -1,6 +1,6 @@
 # Mellanox SAI
 
-MLNX_SAI_VERSION = SAIRel1.19.5
+MLNX_SAI_VERSION = SAIRel1.20.0
```

Comment:
@raphaelt-nvidia  this is not clear to me, This PR doesn't update or upgrade the SAI version. So why to change this file ?

Reply:
Because this version supports SPC4.

Reply:
But SAI 1.20.0 was not upstreamed yet to Master. So please remove this one, it will fail the build.

Reply:
But I think if we compile with a SAI that doesn't support SPC4, it will also fail the build.